bloom-housing/bloom
# Contribute/Update UnitsSummary monthlyRent to rentMin/Max

# Pull Request Template

## Issue

Addresses # 


## Description
As we worked on translating the unitSummarized table to unitsSummary, we noticed unitsSummary only has one monthly rent field.  This change splits that into max and min fields.  The branch has left off with the units-summary-entity update, Detroit seed file refresh, and we've run a migration on the backend.  Next steps are updating backend-swagger.


## Type of change


## How Can This Be Tested/Reviewed?

Open the units_summary table, and view the columns.  You will see monthly rent is now split into a min and max value.


## Checklist:

- [] I have reviewed the changes in a mobile view
